src/main/java/cc/woverflow/hytils/handlers/lobby/armorstands/ArmorStandHider.java

```diff
@@ -29,15 +29,15 @@
 import cc.woverflow.hytils.util.locraw.LocrawInformation;
 
 public class ArmorStandHider {
-    private static final String[] armorStandNames = {"click", "mystery vault", "daily reward tokens", "advent calendar reward", "free rewards", "special holiday quests",
+    private static final String[] armorStandNames = {"click", "mystery vault", "daily reward tokens", "advent calendar reward", "free rewards", "special holiday quests", "release",
         "festive floors", "museums", "hype", "coming soon", " set #", "fireball/tnt jumping", "parkour starts this way", "go ahead into the cave", "holiday mode", "new update", "new modes & maps",
-        "challenges released", "bug fixes & qol update", "catacombs m7!", "sign posting", "parkour challenge", "your arcade games profile", "crimson isle", "featured", "limited time!", "hytale trailer",
-        "happy easter", "happy halloween", "happy holidays", "easter event", "holiday event", "halloween event", "summer event"};
+        "bug fixes & qol update", "sign posting", "parkour challenge", "your arcade games profile", "crimson isle", "featured", "limited time", "hytale trailer",
+        "happy easter", "happy halloween", "happy holidays", "easter event", "holiday event", "halloween event", "summer event", "goal", "defend", "jump in to score"};
 
     @SubscribeEvent
     public void onEntityRenderer(RenderLivingEvent.Pre<EntityLivingBase> event) {
         final LocrawInformation locraw = HytilsReborn.INSTANCE.getLocrawUtil().getLocrawInformation();
-        if (HytilsConfig.hideUselessArmorStands && EssentialAPI.getMinecraftUtil().isHypixel() && HytilsReborn.INSTANCE.getLobbyChecker().playerIsInLobby() || (HytilsConfig.hideUselessArmorStandsSkyblock && locraw != null && locraw.getGameType() == GameType.SKYBLOCK)) {
+        if (HytilsConfig.hideUselessArmorStands && EssentialAPI.getMinecraftUtil().isHypixel() && HytilsReborn.INSTANCE.getLobbyChecker().playerIsInLobby() || (HytilsConfig.hideUselessArmorStandsGame && locraw != null && locraw.getGameType() == GameType.SKYBLOCK) || (HytilsConfig.hideUselessArmorStandsGame && locraw != null && locraw.getGameType() == GameType.BED_WARS) || (HytilsConfig.hideUselessArmorStandsGame && locraw != null && locraw.getGameType() == GameType.SKY_WARS) || (HytilsConfig.hideUselessArmorStandsGame && locraw != null && locraw.getGameMode().contains("BRIDGE"))) {
```

Comment:
```suggestion
		if (EssentialAPI.getMinecraftUtil().isHypixel() && locraw != null && (HytilsReborn.INSTANCE.getLobbyChecker().playerIsInLobby() && HytilsConfig.hideUselessArmorStands) || (HytilsConfig.hideUselessArmorStandsGame && (locraw.getGameType() == GameType.SKYBLOCK || locraw.getGameType() == GameType.BED_WARS || locraw.getGameType() == GameType.SKY_WARS || locraw.getGameMode().contains("BRIDGE"))) {
```
Did this in Notepad++ so not 100% sure if it's right, but I think it is. Regardless of where you are on Hypixel, you *should* always have a locraw, so might as well check that in the beginning and not have to check it for each individual locraw check. Then we can just group up the config option and all possible game(types/modes) together.